test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.CheckViewData.AtViewModel.html

```diff
@@ -0,0 +1,54 @@
+<div class="row">
+    <h4>At Model index</h4>
+    <div class="col-md-3">MetadataKind: 'Type'</div>
+    <div class="col-md-3">ModelType: 'SuperViewModel'</div>
+</div>
+
+<div class="row">
+    
+<h5>Template for ViewModel</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'SuperViewModel'</div>
+
+</div>
+<div class="row">
+    
+<h5>Partial for ViewModel</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'SuperViewModel'</div>
+
+</div>
+<div class="row">
+    <h5>Check View Data view component</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'SuperViewModel'</div>
+
+<h5>Check View Data view component's view</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'SuperViewModel'</div>
```

Comment:
Without `src` changes, get `ModelType: 'ViewModel'` here. Minor case -- got metadata from the declared type rather than the instance.